src/Extensions/Hosting.Services/HostBuilderExtensions.cs

```diff
@@ -25,57 +26,53 @@ public static class HostBuilderExtensions
+			this ServiceFabricHostBuilder<TService> builder,
+			Func<TService,CancellationToken, Task> action) =>
+				builder.AddServiceAction(new ServiceAction<TService>(action));
 
 		/// <summary>
 		/// Add actions that will be executed inside stateless service RunAsync method
 		/// </summary>
-		public static ServiceFabricHostBuilder<TContext> AddServiceAction<TContext>(
-			this ServiceFabricHostBuilder<TContext> builder,
-			IServiceAction<TContext> action)
-				where TContext : ServiceContext =>
-					builder.ConfigureServices((config, collection) => collection.AddSingleton(action));
+		public static ServiceFabricHostBuilder<TService> AddServiceAction<TService>(
+			this ServiceFabricHostBuilder<TService> builder,
+			IServiceAction<TService> action) =>
+				builder.ConfigureServices((config, collection) => collection.AddSingleton(action));
 
 		/// <summary>
 		/// Add service listener to stateless service fabric service
 		/// </summary>
-		public static ServiceFabricHostBuilder<TContext> AddServiceListener<TContext>(
-			this ServiceFabricHostBuilder<TContext> builder,
+		public static ServiceFabricHostBuilder<TService> AddServiceListener<TService>(
+			this ServiceFabricHostBuilder<TService> builder,
 			string name,
-			Func<TContext, ICommunicationListener> createListener)
-				where TContext : ServiceContext =>
-					builder.AddServiceListener(new ListenerBuilder<TContext>(name, createListener));
+			Func<TService, ICommunicationListener> createListener) =>
+				builder.AddServiceListener(new ListenerBuilder<TService>(name, createListener));
 
 		/// <summary>
 		/// Add service listener to stateless service fabric service
 		/// </summary>
-		public static ServiceFabricHostBuilder<TContext> AddServiceListener<TContext>(
-			this ServiceFabricHostBuilder<TContext> builder,
-			IListenerBuilder<TContext> listenerBuilder)
-				where TContext : ServiceContext =>
-					builder.ConfigureServices((config, collection) => collection.AddSingleton(listenerBuilder));
+		public static ServiceFabricHostBuilder<TService> AddServiceListener<TService>(
+			this ServiceFabricHostBuilder<TService> builder,
+			IListenerBuilder<TService> listenerBuilder) =>
+				builder.ConfigureServices((config, collection) => collection.AddSingleton(listenerBuilder));
 
 		/// <summary>
 		/// Configures host to run service fabric stateless service with initializded Omex dependencies
 		/// </summary>
 		public static IHost BuildStatelessService(
 			this IHostBuilder builder,
 			string serviceName,
-			Action<ServiceFabricHostBuilder<StatelessServiceContext>> builderAction) =>
-				builder.BuildServiceFabricService<OmexStatelessServiceRunner, StatelessServiceContext>(serviceName, builderAction);
+			Action<ServiceFabricHostBuilder<StatelessService>> builderAction) =>
+				builder.BuildServiceFabricService<OmexStatelessServiceRunner,StatelessService,StatelessServiceContext>(serviceName, builderAction);
```

Comment:
>StatelessService [](start = 65, length = 16)

Here and on the equivalent line below, leave a space after `,` as per out coding conventions to enhance readability

Reply:
Fixed, thank you